package.json

```diff
@@ -92,9 +92,9 @@
     "@nebula.js/cli-sense": "4.4.0",
     "@nebula.js/cli-serve": "4.4.0",
     "@playwright/test": "1.37.1",
-    "@qlik-oss/nebula-table-utils": "1.1.0",
     "@qlik-trial/sprout": "3.4.0",
     "@qlik/eslint-config": "0.4.15",
+    "@qlik/nebula-table-utils": "^2.0.0",
```

Comment:
Pin the version

Reply:
Yes, pls, then it should be LGTM 🚀
Also sn-table needs the same PR as well otherwise it's CI may fail in installation step i guess (ot it still will continue to use the existing old pkg from github registry)

Reply:
Done

It is true that sn-table needs the same update but I havn't removed anything so nothing will break there